src/main/java/com/robosh/ejournal/service/AdminService.java

```diff
@@ -17,20 +20,39 @@
 
     private final AdminRepository adminRepository;
     private final AdminMapper adminMapper;
+    private final ModelMapper modelMapper;
 
-    public AdminInfoDto save(UpdateAdminDto updateAdminDto) {
-        if (!updateAdminDto.getPassword().equals(updateAdminDto.getConfirmedPassword())) {
+    public AdminInfoDto save(SaveAdminDto saveAdminDto) {
+        if (!saveAdminDto.getPassword().equals(saveAdminDto.getConfirmedPassword())) {
             throw new ValidationException("Password should be same");
         }
-        Admin admin = adminMapper.fromUpdateAdminDtoToAdmin(updateAdminDto);
-        if (updateAdminDto.getId() == null) {
+        Admin admin = adminMapper.fromSaveAdminDtoToAdmin(saveAdminDto);
+        if (saveAdminDto.getId() == null) {
             admin.setSchool(null);
         }
         adminRepository.save(admin);
         return adminMapper.fromAdminToAdminInfoDto(admin);
     }
 
-    public List<AdminInfoDto> getAllAdmins() {
+    public AdminInfoDto update(SaveAdminDto updateAdminDto) {
```

Comment:
you better add new methods to the bottom of file otherwise it may cause merge conflict

Reply:
but then the structure of the class will be messed up